code/modules/projectiles/guns/energy/special.dm

```diff
@@ -317,6 +317,116 @@
 	zoom_amt = 7 //Long range, enough to see in front of you, but no tiles behind you.
 	shaded_charge = 1
 
+/obj/item/gun/energy/bsg
+	name = "\improper B.S.G"
+	desc = "The Blue Space Gun. Uses a flux anomaly core and a bluespace crystal to produce destructive bluespace energy blasts, inspired by Nanotrasens BSA division."
+	icon_state = "BSG"
+	item_state = "BSG"
+	origin_tech = "combat=6;materials=6;powerstorage=6,bluespace=6;magnets=6" //cutting edge technology, be my guest if you want to deconstruct one instead of use it.
```

Comment:
```suggestion
	origin_tech = "combat=6;materials=6;powerstorage=6;bluespace=6;magnets=6" //cutting edge technology, be my guest if you want to deconstruct one instead of use it.
```